app/views/moderator_applications/new.html.slim

```diff
@@ -0,0 +1,10 @@
+.container-fluid
+  .row
+    .col-lg-2
+
+    main.col-lg-8
+      h1 New Moderator Application
+      = render 'responsibilities'
+      = render 'form', application: @application, over_18: false
```

Comment:
`over_18: false` should be the default value, but I didn't know how to set a default value when I wrote this. Though really, this should probably be part of the application model in the first place, since it *is* data about the application, even if it is implicit; a model which has already been saved should be pre-checked, because you can only save the application in the first place if you have already confirmed your age. Either that, or I make it a database table row, and allow under-18 users to submit applications (even if they may not be accepted)?

Also, `over_18` isn't a very descriptive variable name.

Reply:
generally speaking, convention for boolean symbol names are declarative and descriptive, so in this case "is_over_18" would probably be more prudent.
however this is minutiae that i literally don't think merits stressing out over.